### PR TITLE
Included appveryor file for CI with windows

### PR DESCRIPTION
**Pull request by jrbeaumont**
For #3, Windows integration with Appveyor. 

https://ci.appveyor.com/project/jrbeaumont/process-mining

Builds successfully.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - 7z x cabal.tar.gz
+  - 7z x cabal.tar
+  - ren cabal-1.22.0.0-i386-unknown-mingw32.exe cabal.exe
+
+
+test_script:
+  - stack init
+  - stack setup > nul
+  - echo "" | stack --no-terminal build
+  - stack test
```
Comment (snowleopard) [inline appveyor.yml]: Github is worried by the lack of newline here -- please add one :)

**Comment by snowleopard** [comment]
In addition to the nit-picky comment above, could you print out the current GHC version? That is often useful. I think something like this at the end of `before_test` should work:

```
stack exec -- ghc --version
```

**Comment by jrbeaumont** [comment]
Sure thing :)

**Comment by jrbeaumont** [comment]
https://ci.appveyor.com/project/jrbeaumont/process-mining

Included the ghc version in the build. Now correctly builds :)

**Comment by snowleopard** [comment]
:+1:

**Pull request merged**